scss/_forms.scss

```diff
@@ -28,7 +28,7 @@ textarea {
   vertical-align: middle;
   background-color: #fff;
   background-repeat: no-repeat; // Repeat and position set for form states (success, error, etc)
-  background-position: right center;
+  background-position: right 8px center;
```

Comment:
I thought this was a little too clever, but up to four values are totally valid. Old declarations throughout .com are 98% from the left, but this is more consistent and renders more accurately regardless of the size of the input. This way it'll always be 8px from the right and centered vertically.

https://css-tricks.com/almanac/properties/b/background-position/#browser-support